### ISSUE TITLE
Add meta data related to momentum calibration (to NXcalibration)

### ISSUE THREAD

**Issue by rettigl** [Arora0/nomad-parser-nexus#3]

**Comment by Arora0** [comment]
I am unsure about what more can be added to the momentum calibration from the mc class dict. Same with issues 4 and 6. Distortion class fields are all filled and that issue has been closed.

**Comment by rettigl** [comment]
Have a look in the binning notebook how the image registration and the momentum calibration are done. These information should be stored. If this is not part of the class dictionaries, then create new structures for it. For the registration, we could in principle also use NXtransformation classes.

**Comment by rettigl** [comment]
I added a notebook to the examples for binning from the raw data files. This currently breaks at the entry that you added for the momentum calibration coefficients (wrong data type). Please have a look and fix this if possible.

**Comment by Arora0** [comment]
Fixed the issue and also added path for the config file since I thought that it will generally not be present in the uploads but in the cloned repo. The h5web visualization works fine. I had an import error for nexusparser (cannot import Bytes in nexus.py) which was resolved when I ran the nomad installation step in step 0 despite it being within nomad. 
Also, in NOMAD, on reprocessing the upload, it correctly identifies the nexus output file WSe2.mpes.nxs but the status shows 'failed'. The following entry page also shows the correct nexus tree albeit with a problem in the default plot. Don't know if it's supposed to work already. I've shared the upload with you.

**Comment by rettigl** [comment]
Ah, now I understand what you mean. Yes, the nexustools container does not have the nomad package installed per default, that's why there is the step 0 in the script.

**Comment by Arora0** [comment]
Ah, I see. Is it then installed in some another container at the moment?

**Comment by rettigl** [comment]
No, I don't think so, but you can just install it with the commands in the notebook
